crates/mxd-verification/Dockerfile

```
@@ -0,0 +1,14 @@
+# Dockerfile for running TLC model checker
+# Uses the latest TLA+ tools from GitHub releases
+
+FROM docker.io/eclipse-temurin:21-jre-alpine
+
+# Download the latest tla2tools.jar
+RUN apk add --no-cache curl && \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)